TunnelManager/Session.cs

```diff
@@ -229,6 +237,7 @@ public virtual void Serialize()
                 RegistryKey ptmSessionKey = Registry.CurrentUser.CreateSubKey(this.PuttyTunnelManagerKeyPath);
                 ptmSessionKey.SetValue(PTM_REGISTRY_KEY_SESSION_PORTFORWARDINGS, buffer.ToString(), RegistryValueKind.String);
                 ptmSessionKey.SetValue(PTM_REGISTRY_KEY_SESSION_USEPTMFORTUNNELS, this.usePtmForTunnels, RegistryValueKind.DWord);
+                ptmSessionKey.SetValue(PTM_REGISTRY_KEY_SESSION_AUTOSTART, this.autoStart, RegistryValueKind.DWord);

```

Comment:
If sessions not stored in PMT - autostart not available? If so "autostart" option must be disabled. In other case autostart function must work Independent.